### ISSUE TITLE
[Feature] Headless

### ISSUE THREAD

**Issue by jenslys** [vasani-arpit/Youtube-remove-copyright#11]
Hi!
Would it be possible to configure this to run headless?

**Comment by vasani-arpit** [comment]
Yes,  It is possible. Might need to add something like stealth mode from [puppteer-extra](https://github.com/berstend/puppeteer-extra/tree/master/packages/puppeteer-extra) to prevent detection. I think.
